### PR TITLE
Add multi repo support, comma separated

### PR DESCRIPTION
**Pull request by aadityataparia**

### CODE REVIEW

File: lib/mergometer/report.rb [hunk 1]
```diff
@@ -46,13 +46,19 @@ def entries
...
       end
 
       def fields_to_preload
         []
       end
 
+      def repo_query
+        @_query = repo.split(',').map do |r|
+          "repo:#{r} "
+        end.to_a.join
```
Comment (balvig) [inline lib/mergometer/report.rb]: Map already return an array I guess 😉

File: lib/mergometer/report.rb [hunk 2]
```diff
@@ -46,13 +46,19 @@ def entries
...
-        "base:master repo:#{repo} type:pr is:merged"
+        "base:master #{repo_query} type:pr is:merged"
       end
 
       def fields_to_preload
         []
       end
 
+      def repo_query
+        @_query = repo.split(',').map do |r|
```
Comment (balvig) [inline lib/mergometer/report.rb]: Should we use the same naming for the method / memoised variable?

(Nit: double quotes 😉)

File: lib/mergometer/report.rb (hunk 1 above)
Comment (aadityataparia) [reply lib/mergometer/report.rb]: Enumerator :(

File: lib/mergometer/report.rb (hunk 2 above)
Comment (aadityataparia) [reply lib/mergometer/report.rb]: Oops

**Comment by aadityataparia** [comment]
@balvig ♻️

File: lib/mergometer/report.rb (hunk 1 above)
Comment (balvig) [reply lib/mergometer/report.rb]: Can you double check?

```ruby
irb(main):002:0> ["1", "2"].map { |v| v }.join
=> "12"
```

**Comment by balvig** [comment]
> Could you check whether the other reports support this? Some of them use custom filters so may be necessary to use repo_query there as well!

Could you check this too? 🙇 🙏

File: lib/mergometer/report.rb (hunk 1 above)
Comment (aadityataparia) [reply lib/mergometer/report.rb]: You are right, it seems I was using old ruby version before 🙇

**Comment by aadityataparia** [comment]
@balvig ♻️

File: lib/mergometer/report.rb [hunk 3]
```diff
@@ -46,13 +46,19 @@ def entries
...
+        "base:master #{repo_query} type:pr is:merged"
       end
 
       def fields_to_preload
         []
       end
 
+      def repo_query
+        @_repo_query = repo.split(",").map do |r|
+          "repo:#{r} "
```
Comment (balvig) [inline lib/mergometer/report.rb]: Nit: is there an extra space here? Could maybe add it to `join(" ")` instead?

Comment (aadityataparia) [reply lib/mergometer/report.rb]: Done! @balvig

**Pull request merged**